frontend/plugin-manager/src/components/plugin/hosted/ui-kit/runtime.test.ts

```
@@ -0,0 +1,435 @@
+    i18n: {
+      locale: 'en',
+      default_locale: 'en',
+      messages: {
+        en: {
+          greeting: 'Hello {name}',
+        },
+      },
+    },
+  }
+  window.NekoUiKit = undefined
+  window.eval(runtimeSource)
```

Biome (2.4.13)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)